### PR TITLE
Created Slackers custom theme, added search bar component to it

### PR DESCRIPTION
**Pull request by philliparaujo**
This PR adds a new custom theme in our root directory called `nodebb-theme-slackers`. In this PR, I also used this theme to create a new search bar component with basic listener functionality (i.e. typing and clicking on it prints to the console).

I also updated our README to explain how to use our new custom theme once it's merged to main.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Test
+
```
Comment (KesterTan) [inline README.md]: Should we remove this

File: nodebb-theme-slackers/languages/harmony.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (KesterTan) [inline nodebb-theme-slackers/languages/harmony.json:1]: Not sure if we need this

**Review by KesterTan — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (philliparaujo) [reply README.md]: Good catch

**Pull request merged**